modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/components/Radio/index.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import ClayCard from '@clayui/card';
+import {ClayRadio} from '@clayui/form';
+import classNames from 'classnames';
+import {forwardRef} from 'react';
+
+export const Radio = forwardRef(
+	(
+		{
+			description,
+			label,
+			name,
+			onChange,
+			renderActions,
+			selected = false,
+			sideLabel,
+			value,
+			...props
+		},
+		ref
+	) => {
+		return (
+			<ClayCard
+				className={classNames(
+					'align-items-baseline d-flex flex-row mb-3 py-2 py-sm-3 px-sm-3 px-2 radio-card rounded user-select-auto',
+					{
+						'bg-brand-primary-lighten-5 border border-primary text-brand-primary': selected,
```

Comment:
Don't border-primary and border do the same thing? We really need both?

Reply:
You're right! Thank you!